db/migrate/20220718092743_create_task_lists.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTaskLists < ActiveRecord::Migration[7.0]
+  def change
+    create_table :task_lists do |t|
+      t.integer :task_count
```

Comment:
Is `task_count` something you want to have in the database?
Updating such a record would cause some performance issues - I mean in app used by hundreds of users.
Plus you never now how many users will access and updates same record at one time.

Usually what you could do is call method `TaskList.taks.count` to get the actual number of tasks in the list